subprojects/win-iconv.wrap

```diff
@@ -0,0 +1,10 @@
+[wrap-file]
+directory = win-iconv-0.0.8
+source_url = https://github.com/win-iconv/win-iconv/archive/refs/tags/v0.0.8.tar.gz
+source_filename = win-iconv-0.0.8.tar.gz
+source_hash = 23adea990a8303c6e69e32a64a30171efcb1b73824a1c2da1bbf576b0ae7c520
+patch_directory = win-iconv
+
+[provide]
+iconv = win_iconv_dep
```

Comment:
Minor: since `meson.build` unconditionally calls `meson.override_dependency()`, this can be written as

```
dependency_names = iconv
```

Reply:
yes, but then we need another meson version check, as `meson.override_dependency()` was introduced in 0.60.0 ish (I have to look it up, but it was before 1.0)

Reply:
It was introduced in `>=0.54.0` So I'll use that as meson version, which seems reasonable to me, since that is a really old version, which everybody should be able to use 👍🏼